### PR TITLE
feature CSI-3610 eLicense 1.8.0

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

**Review by erantzabari — APPROVED** [review]

File: NOTICES [hunk 1]
```diff
@@ -2411,19 +2411,23 @@ APACHE 2.0 LICENSED CODE:  The Program includes all or portions of
...
 controller-tools version 0.1.11
-controller-util version 0.2.6
+controller-util commit fc080f9
 gengo commit e17681d
+grpc version 1.27.0
+kube-openapi version v0.0.0-20200121204235-bf4fb3bd569c
 logr version 0.1.0
-operator-sdk version 1.9.0
-yaml version 1.2.0
+operator-sdk version 0.18.1
```
Comment (oriyarde) [inline NOTICES:2427]: why were all the dependencies downgraded here?

Comment (ArbelNathan) [reply NOTICES:2427]: IDK, its 1.9.0 in the OSSC

Comment (erantzabari) [reply NOTICES:2427]: I will raise it to legal department

Comment (oriyarde) [reply NOTICES:2427]: thanks

**Pull request merged**